docs/adguard-for-windows/admins-documentation.md

```diff
@@ -39,12 +40,16 @@ If you want to install AdGuard on a Windows 7 computer, make sure that it has .N
 
 :::info
 
-Starting with AdGuard v7.17 for Windows, automatic updates are enabled by default when you install the application from MSI. We recommend that you disable updates for a user so that AdGuard for Windows is updated centrally. To do this, add the ForbidCheckUpdates=Yes key, which is located at the following path: HKEY_LOCAL_MACHINE\SOFTWARE\Adguard.
+When installing AdGuard for Windows from MSI, AdGuard is not automatically updated. If you want to allow updates for the user (which is not recommended, because it prevents centralized updates), you must either delete the `ForbidCheckUpdates` parameter or set its value to `NO` (not case-sensitive) in the registry under `HKEY_LOCAL_MACHINE\SOFTWARE\Adguard`. Doing so will permit updates.
```

Comment:
```suggestion
When installing AdGuard for Windows from MSI, AdGuard is not automatically updated. If you want to allow updates for the user (which is not recommended, because it prevents centralized updates), you must either delete the `ForbidCheckUpdates` parameter or set its value to `NO` (case-insensitive) in the registry under `HKEY_LOCAL_MACHINE\SOFTWARE\Adguard`. This will allow updates.
```

Reply:
что-то я не улавливаю: «При установке AdGuard для Windows из MSI AdGuard не обновляется автоматически. Если вы хотите разрешить обновления для пользователя (что не рекомендуется, поскольку это препятствует централизованным обновлениям), вы должны...». в первом предложении мы говорим, что адгард не обновляется автоматически (системой), а во втором говорим: «если вы хотите разрешить обновления для пользователя (что препятствует централизованным обновлениям — отключенным)... непонятно

Reply:
в общем тут тема такая. Если пользователь установил наш exe, то автоматически приходят обновления и всё хорошо. Если же это гиковская установка через msi (например, на предприятии, чтобы сделать централизованно), то это самое автоматическое обвновление отключено (по многим причинам). 

>Если вы хотите разрешить обновления для пользователя (что не рекомендуется, поскольку это препятствует централизованным обновлениям)


Но мы должны предложить вариант пользователю наплевать на нас и всё равно обновляться (на свой страх и риск - там на самом деле есть тезнические проблемы)



>что препятствует централизованным обновлениям

в том смысле, что эти самые централизованные обновления будут централизованно идти от местного сисадмина

Reply:
тогда предлагаю так переписать:

Если вы установили AdGuard для Windows из файла MSI, AdGuard не будет обновляться автоматически. Чтобы разрешить обновления для пользователя, вы должны удалить параметр `ForbidCheckUpdates` или установить его значение `NO` (регистр не важен) в реестре под `HKEY_LOCAL_MACHINE\SOFTWARE\Adguard`. Но обратите внимание, что такая установка приложения и ручные обновления влекут за собой технические проблемы и не рекомендуются.